it/test/controllers/errors/SupportingAgentAuthErrorControllerISpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This whole file should probably be a unit test it should be refactored into test/controllers/errors

See other PR's